### PR TITLE
Explicitly use UTF-8 encoding.

### PR DESCRIPTION
**Pull request by ehrlichja**

### CODE REVIEW

File: camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java [hunk 1]
```diff
@@ -62,7 +63,12 @@ public void init(Properties props, String topicName) {
     String payloadString;
     JsonObject jsonObject;
 
-    payloadString = new String(payload);
```
Comment (HenryCaiHaiying) [inline camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java]: Does Kafka always encode bytes in UTF-8?  Do they configure the encoding in one of the kafka config parameter?

File: camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java [hunk 2]
```diff
@@ -62,7 +63,12 @@ public void init(Properties props, String topicName) {
...
+    try {
+      payloadString = new String(payload, "UTF-8");
+    } catch (UnsupportedEncodingException e) {
+      log.error("Unable to load UTF-8 encoding, falling back to system default", e);
```
Comment (erikvanoosten) [inline camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java]: UTF-8 is a required encoding according to the the JVM specification. Therefore, IMHO this much ceremony is not needed.

File: camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java (hunk 1 above)
Comment (erikvanoosten) [inline camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/JsonStringMessageDecoder.java]: Only a few encodings are allowed for JSON: UTF-8, UTF-16 and UTF-32 if memory serves.
Therefore: JSON is pretty much always in UTF-8.

**Comment by aehrlich-unifiedsocial** [comment]
I added this because a camus/EMR setup was converting UTF-8 characters into "???". 

Close, merge, or update?

**Comment by ehrlichja** [comment]
I added this because a camus/EMR setup was converting UTF-8 characters into "???".

Close, merge, or update?

**Pull request merged**